### PR TITLE
sql-statement: add ADMIN ddl statements (#4801)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #4801 to release-3.0

---

### What is changed, added or deleted? (Required)
First batch of https://github.com/pingcap/docs/issues/2639

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/3587 https://github.com/pingcap/docs/pull/4184
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [x] Might cause conflicts

### CODE REVIEW

**Comment by Joyinqin** [comment]
@lilin90 PTAL, thanks

File: sql-statements/sql-statement-admin-check-table-index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+以上语句用于对 `tbl_name` 表中 `idx_name` 索引对应列数据和索引数据进行一致性校验，并且指定了需要检查的数据范围。若通过校验，则返回空的查询结果；否则返回数据不一致的错误信息。
+
+## MySQL 兼容性
+
+`ADMIN CHECK [TABLE|INDEX]` 语句是 TiDB 对 MySQL 语法的扩展。
+
+## 另请参阅
+
+* [`ADMIN REPAIR`](/sql-statements/sql-statement-admin.md)
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin-check-table-index.md:54]:
```suggestion
* [`ADMIN`](/sql-statements/sql-statement-admin.md)
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-admin-cancel-ddl.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+> **注意：**
+>
+> + 只有该操作可以取消 DDL 作业，其他所有的操作和环境变更（例如机器重启、集群重启）都不会取消 DDL 作业。
+> + 该操作可以同时取消多个 DDL 作业，可以通过 `ADMIN SHOW DDL JOBS` 语句来获取 DDL 作业的 `job_id`。
+> + 如果希望取消的作业已经执行完毕，取消操作将失败。
+
+## MySQL 兼容性
+
+`ADMIN CANCEL DDL` 语句是 TiDB 对 MySQL 语法的扩展。
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin-cancel-ddl.md:40]:
```suggestion
`ADMIN CANCEL DDL` 语句是 TiDB 对 MySQL 语法的扩展。 
```

File: sql-statements/sql-statement-admin-cancel-ddl.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin-cancel-ddl.md:40]:
```suggestion
`ADMIN CANCEL DDL` 语句是 TiDB 对 MySQL 语法的扩展。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**